ui/webui/test/helpers/installer.py

```diff
@@ -50,23 +55,25 @@ def begin_installation(self, should_fail=False, confirm_erase=True):
             self.browser.click(".pf-c-modal-box button:contains(Back)")
 
         if should_fail:
-            self.wait_current_page(self.steps[current_step_id])
+            self.wait_current_page(current_page)
         else:
-            self.wait_current_page(self.steps[current_step_id+1])
+            self.wait_current_page(self.steps._steps_jump[current_page])
 
     @log_step()
     def next(self, should_fail=False, subpage=False, next_page=""):
-        current_step_id = self.get_current_page_id()
-        current_page = self.steps[current_step_id]
-        next_page = next_page or self.steps[current_step_id+1]
+        current_page = self.get_current_page()
+        # If not explicitly specified, get the first item for next page from the steps dict
+        next_page = next_page or self.steps._steps_jump[current_page][0] if type(self.steps._steps_jump[current_page]) is list else self.steps._steps_jump[current_page]
```

Comment:
Maybe making this a normal `if and else` will be a lot more readable then decoding this shorthand if.